introduction_to_amazon_algorithms/jumpstart-foundation-models/domain-adaption-finetuning-gpt-j-6b.ipynb

```diff
@@ -0,0 +1,782 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    use_amt = False
</pre><p>We generally set it to True in other JS notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3899/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>It will run too long, and cause time out of 2 hours limit. I tested it offline it works well.</p>